### PR TITLE
Add the "financiering" table

### PR DESCRIPTION
**Pull request by Windvis**
This adds the "financiering" section to the urban renewal subsidy with the first table.

Please double check the predicates I used. I'm not sure if these are the best names for them (or if they follow the correct conventions).

### CODE REVIEW

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl [hunk 1]
```diff
@@ -480,8 +482,105 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
...
+    sh:group ext:financingPropertyGroup .
+
+##########################################################
+# Field: Bedrag
+##########################################################
+ext:financingAmountF a form:Field ;
+    sh:name "Bedrag" ;
+    sh:order 3;
+    sh:path ext:financingAmount;
+    form:displayType displayTypes:numericalInput ;
```
Comment (Windvis) [inline config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl:577]: This needs to be replaced with something better, but I don't think we have a "currency" field yet?

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json [hunk 2]
```diff
@@ -14,13 +14,23 @@
...
             "schema:jobTitle"
           ]
         },
+        {
+          "s-prefix": "ext:financingPartner",
+          "properties": [
+            "rdf:type",
+            "ext:partnerTitle",
+            "ext:partnerType",
+            "ext:financingAmount"
```
Comment (Windvis) [inline config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json:31]: The `shacl:order` needs to be added here as well

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl (hunk 1 above)
Comment (aatauil) [reply config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl:577]: I dont think so, I also just used numericalInput

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json [hunk 3]
```diff
@@ -14,13 +14,23 @@
...
             "foaf:familyName",
             "schema:telephone",
             "schema:email",
             "schema:jobTitle"
           ]
         },
+        {
+          "s-prefix": "ext:financingPartner",
+          "properties": [
+            "rdf:type",
```
[the remark this answers is not in the document]
Comment (Windvis) [reply config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json:28]: Felix added that (I squashed the commits), apparently it was needed for the data fetching to work (see rocket.chat thread).

[the remark this answers is not in the document]
Comment (aatauil) [reply config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json:28]: :+1:

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json (hunk 2 above)
Comment (aatauil) [reply config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json:31]: Dont know what this refers to

**Review by aatauil — APPROVED** [review]
Besides the shacl comment, rest looks good

File: config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json (hunk 2 above)
Comment (aatauil) [reply config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.json:31]: Yeah probably, I think I might have the same issue in projectonderdelen. Will add the shacl:order to the list for you

**Pull request merged**